languages/elixir/exercises/concept/tuples/.docs/introduction.md

```diff
@@ -0,0 +1,67 @@
+In Elixir, a tuple is a data structure which organizes data in contiguous memory. It behaves as a static data structure, holding a fixed number of items, but without explicit names for each element. Tuples are often used in Elixir for memory read-intensive operations, since read-access of an element is a constant-time operation. They are not usually used when elements may need to be added/removed dynamically because rather than modifying the existing tuple, a new tuple is created which requires memory to be allocated upfront.
```

Comment:
Just for your consideration: I was wondering if the memory aspect of tuples is something to mention in the `introduction.md` file, or if that is possibly more suited for the `after.md` document?

Reply:
Maybe I could shorten this for brevity and keep some element of the memory features here.

Reply:
Yep, that sounds like a plan! That also matches what @angelikatyborska suggested.